dist/cmd-resume.js

```diff
@@ -1112,10 +1112,6 @@
   
   		self.settings = {
   			greetings: self.commands.splash.handler(),
-  			onBlur: function() {
```

Comment:
No substitute?

Reply:
Why? This was a part of the snippet on how to embed years ago, not required anymore. Regardless it isn’t mobile friendly.